smurfs/src/components/SmurfForm.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react'
+import axios from 'axios'
+
+const SmurfForm = () => {
+    const [name, setName] = useState()
+    const [age, setAge] = useState()
+    const [height, setHeight] = useState()
+
```

Comment:
Instead of individually creating state for name, age, and height, you could do this:
```suggestion
const [smurf, setSmurf] = useState({
    name: name,
    age: age,
    height: height,
    id: id
})
```